common/Slash.yml

```
@@ -0,0 +1,21 @@
+extends: existence
+message: "Only use slashes with filenames, links and terms such as TCP/IP."
```

Please rephrase to add the %s somewhere, because without it's very difficult to spot the issue

```suggestion
message: "%s: Only use slashes with filenames, links and terms such as TCP/IP."
```

---

@alexvonme you resolved, but I don't see changes on the PR.

Did I miss something?

---

Should be better now.